### PR TITLE
Unit tests #2

### PR DESCRIPTION
**Pull request by Azrael-git**
1. Add many tests (more than 99%)
2. Fixed spaces
3. Fixed logic
4. Delete trash code
Trello card: https://trello.com/c/RghxTHTW/62-unit-tests-coverage-for-all-updates

### CODE REVIEW

File: api/authentication/models.py [hunk 1]
```diff
@@ -1,10 +1,21 @@
...
 from django.core.mail import send_mail
 from django.db import models
 from django.utils import timezone
+from django.core.signing import TimestampSigner, b64_encode,b64_decode, BadSignature, SignatureExpired,force_bytes
+from django.core.mail import EmailMessage
 from django.contrib.auth.models import UserManager
-from django.contrib.auth.models import AbstractUser, Group
+from django.contrib.auth.models import AbstractBaseUser, AbstractUser, Group
+
+from settings import base
```
Comment (pavlo-mk) [inline api/authentication/models.py]: the same issue this import settings. dont use this import in future

**Review by pavlo-mk — CHANGES_REQUESTED** [review]
Please make updates and merge #32 and later I think we can approve this PR

**Comment by pavlo-mk** [comment]
@Azrael-git please make pull from `master` into this branch and reassign me for review.

**Comment by Azrael-git** [comment]
> @Azrael-git please make pull from `master` into this branch and reassign me for review.

Resolved

**Review by pavlo-mk — APPROVED** [review]
LGTM.

**Pull request merged**